FBRetainCycleDetector/Layout/Classes/FBClassStrongLayout.m

```diff
@@ -166,14 +166,26 @@ static NSUInteger FBGetMinimumIvarIndex(__unsafe_unretained Class aCls) {
   return filteredIvars;
 }
 
-NSArray<id<FBObjectReference>> *FBGetObjectStrongReferences(id obj) {
+NSArray<id<FBObjectReference>> *FBGetObjectStrongReferences(id obj,
+                                                            NSMutableDictionary<Class, NSArray<id<FBObjectReference>> *> *layoutCache) {
```

Comment:
Actually `Class` does not conform to `<NSCopying>`, but it does have a `-copyWithZone:` method.